queries/yuck/indents.scm

```diff
@@ -9,7 +9,8 @@
 
 ; TODO: can't get this to work, goal is to indent at the property ":" prefix
 ; ((list (identifier) (property)) @aligned_indent
-;   (#set! "delimiter" ":"))
+;   (#set! "open_delimiter" "(")
+;   (#set! "close_delimiter" ")"))
```

Comment:
Here too
Maybe this works now? Not sure

Reply:
Certainly copy pasta.  I'll address that first and can look at whether it now works separately (I didn't change that comment, so it may or may not)